src/ApiGateway/SimpleWebsocketClient.php

```diff
@@ -0,0 +1,62 @@
+<?php declare(strict_types=1);
+
+namespace Bref\ApiGateway;
```

Comment:
Sorry to nitpick but could be use a `Bref\Websocket` namespace?

I think it would make more sense if later we move this to a separate package, it would probably named `bref/websocket` (it would be specific to websocket).

Reply:
@mnapoli Sure - I'll change this